workers/amazonPayments.js

```diff
@@ -125,7 +125,7 @@ var worker = function(job, done){
 
                 console.log('Cancelling', user._id, user.purchased.plan.customerId, amzRes);
                 request({
-                  url: 'https://habitica.com/amazon/subscribe/cancel',
+                  url: nconf.get("HABITICA_URL")+'/amazon/subscribe/cancel',
```

Comment:
you can use base url from here https://github.com/HabitRPG/habitrpg-email-server/blob/master/index.js#L17